mw-webapp/src/logic/wayPage/WayStatistic.tsx

```diff
@@ -105,9 +105,9 @@ export const WayStatistic = (props: WayStatisticProps) => {
 
   const lastCalendarTwoWeekTotalTime = lastTwoWeekJobsAmount.reduce((totalTime, jobDone) => totalTime + jobDone.time, 0);
 
-  const lastCalendarTwoWeekAverageWorkingTime = Math.round(lastCalendarTwoWeekTotalTime / AMOUNT_DAYS_IN_WEEK);
+  const lastCalendarTwoWeekAverageWorkingTime = Math.round(lastCalendarTwoWeekTotalTime / AMOUNT_DAYS_IN_TWO_WEEK);
```

Comment:
what if there is only 9 days in the report? for example?

Reply:
So? average working time allways about 2 weeks. Even if you just started your way

Reply:
Fixed